packages/legend-extension-external-store-service/src/models/metamodels/pure/model/packageableElements/store/serviceStore/mapping/ServiceRequestBuildInfo.ts

```diff
@@ -15,35 +15,19 @@
  */
 
 import { hashArray, type Hashable } from '@finos/legend-shared';
+import type { ServiceRequestBodyBuildInfo } from './ServiceRequestBodyBuildInfo';
+import type { ServiceRequestParametersBuildInfo } from './ServiceRequestParametersBuildInfo';
 import { SERVICE_STORE_HASH_STRUCTURE } from '../../../../../../../ESService_ModelUtils';
-import { ServiceParameterMapping } from './ServiceParameterMapping';
-import { action, computed, makeObservable, observable } from 'mobx';
 
-export class PropertyIndexedParameterMapping
-  extends ServiceParameterMapping
-  implements Hashable
-{
-  property!: string;
+export class ServiceRequestBuildInfo implements Hashable {
+  requestBodyBuildInfo: ServiceRequestBodyBuildInfo | undefined;
```

Comment:
Let's use `?:` as its more correct, the value sent by engine is optional or `undefined`